features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/components/AddOperation.jsx

```diff
@@ -179,6 +179,14 @@ function AddOperation(props) {
             }));
             return;
         }
+        if (api.type.toLowerCase() === 'websub'
+            && (newOperations.target.indexOf('{') > -1 || newOperations.target.indexOf('}') > -1)) {
```

Comment:
Shall we use a regex for the validation as done in L166 instead of a normal string lookup? Refer [1] for implementation.

[1] https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/APIValidation.js#L123

Reply:
Added in 385965d52a2ebd4e2f5ea7f9a639b796b285a680